webapp/src/styles/utils/_utils.scss

```diff
@@ -41,3 +41,8 @@
 .height-290 {
     height: 290px;
 }
+
+// Padding
+.padding-left-6 {
```

Comment:
I think these should be dynamic. I mean whatever value we provide after `padding-left-`, that should be a variable. Check with a FE dev and see if it is too complex.

Reply:
that is implemented in UI Lib, here I don't think we need too many of such util classes so for 4-5 such classes we can write it this way, but if such utils are frequently used then we can take reference from UI Lib and implement that logic here in `SCSS`

Reply:
discussed with @manojmalik20, we are not implementing this